src/AiLibrary.flix

```diff
@@ -90,33 +86,24 @@ namespace Flixball/AiLibrary {
 
     ///
     /// Selects a move based on the shortest path from position p to a
-    /// coordinate where `goal` is true.
+    /// coordinate where `goal` is true. Considers players obstacles if
+    /// `passablePlayers` is `false`.
     ///
-    def goTo(p: Position, goal: Coordinates -> Bool \ ef, b: Board): Option[Move] \ ef = {
+    def goTo
+            (p: Position, goal: Position -> Bool \ ef, passablePlayers: {passablePlayers=Bool}, b: Board):
```

Comment:
Why's this in a separate line? I haven't seen this style before.

Reply:
To avoid a 120+ textwidth. I know this isn't pretty but its the best I've found

Reply:
Flix really needs a style guide